yearn/package.json

```
@@ -1,9 +1,25 @@
 {
+  "name": "@keep-network/tbtc-v2-yearn",
+  "license": "MIT",
+  "scripts": {
+    "build": "hardhat compile",
+    "deploy": "hardhat deploy --export export.json",
+    "lint": "npm run lint:js && npm run lint:sol",
+    "lint:fix:js": "eslint . --fix",
+    "lint:fix:sol": "solhint 'contracts/**/*.sol' --fix && prettier --write '**/*.sol'",
+    "lint:js": "eslint . ",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettier --check '**/*.sol'",
+    "size-contracts": "hardhat compile && hardhat size-contracts",
+    "test": "hardhat test",
+    "test:system": "NODE_ENV=system-test hardhat test ./test/system/*.test.js"
+  },
   "dependencies": {
     "@keep-network/yearn-vaults": "git://github.com/keep-network/yearn-vaults.git#5f1e829",
```

Can we use `github:` for consistency with other references to github packages?

```suggestion
    "@keep-network/yearn-vaults": "github:keep-network/yearn-vaults.git#5f1e829",
```

---

https://github.com/keep-network/tbtc-v2/pull/14/commits/86b04f0287765fbbce093787d9771e81a78320fb